source/common/ur_params.hpp

```diff
@@ -2013,11 +2023,13 @@ operator<<(std::ostream &os,
     os << (params.stype);
 
     os << ", ";
+
     os << ".pNext = ";
 
     ur_params::serializeStruct(os, (params.pNext));
 
     os << ", ";
+
```

Comment:
This extra spaces seem unecessary to me. I wonder where they are coming from and if it is possible to avoid generating them?